src/helpers/validateMod.js

```diff
@@ -0,0 +1,24 @@
+import { getDefaultMod10Weights, getDefaultMod11Weights} from '.'
+/**
+ * 
+ * @param {number[]} digits all the digits including the control digit
+ * @param {number[]} weights (optional)
+ * @param {(digits: number[], weights: number[]) => number} modulo mod10 or mod11 function
+ * @param {'mod10' | 'mod11'} modType the type of modulo calculation used
+ */
+export default function validate(digits, weights, modulo, modType) {
```

Comment:
Since we don't use this method directly but via `mod11_validate` and `mod10_validate`, we could do without the `modType` parameter.
Instead, we could require that weights are supplied, which would be the job of `mod11_validate` and `mod10_validate` to supply the correct ones.

Reply:
Fixed by b2218b7